recipe/meta.yaml

```diff
@@ -1,24 +1,27 @@
+{% set version = "3.1.0" %}
+
 package:
     name: qutip
-    version: 3.1.0
+    version: {{ version }}
 
 source:
-    fn: qutip-3.1.0.tar.gz
-    url: http://qutip.org/downloads/3.1.0/qutip-3.1.0.tar.gz
+    fn: qutip-{{ version }}.tar.gz
+    url: http://qutip.org/downloads/{{ version }}/qutip-{{ version }}.tar.gz
+    sha256: 078399c9830c011ec189db222fcd45f71d2a135800874784d8386d67b4357402
     patches:
       - qutip__init__.patch
-    
+
 build:
     number: 0
     script: $PYTHON setup.py install
```

Comment:
The thing is `script` is preferred to the `bld.bat` and `build.sh` files. That is why Windows is failing, because that string does not work due to the `$PYTHON`. However, even with that fixed Windows probably won't work (see the `bld.bat`).

So by adding the `build.sh` for `*nix` and removing the script is the only work around I know.